### PR TITLE
Footer: Add svg logos for social icons - Design improvements part of #203

### PR DESCRIPTION
**Pull request by christinach**
part of #203

### CODE REVIEW

**Comment by christinach** [comment]
I will change the facebook logo in a different PR

File: public/assets/logos/LogoFacebook.vue [hunk 1]
```diff
@@ -0,0 +1,57 @@
+<template>
```
Comment (sandbergja) [inline public/assets/logos/LogoFacebook.vue:1]: Is this component used?

**Review by sandbergja — COMMENTED** [review]
Thanks, @christinach 🐬 .  Just a question about the facebook component

File: public/assets/logos/LogoFacebook.vue (hunk 1 above)
Comment (christinach) [reply public/assets/logos/LogoFacebook.vue:1]: oh I added it and didnt use it. I will do this in a later work. Thanks I will remove it.

**Review by sandbergja — APPROVED** [review]
🐬

**Pull request merged**